tests/test_rendering.py

```diff
@@ -0,0 +1,42 @@
+async def test_rendering(lona_project_context):
+    import os
+
+    from playwright.async_api import async_playwright
+
+    from lona.html import HTML
+
+    TEST_PROJECT_PATH = os.path.join(__file__, '../../test_project')
+
+    context = await lona_project_context(
+        project_root=TEST_PROJECT_PATH,
+        settings=['settings.py'],
+    )
+
+    async with async_playwright() as p:
+        browser = await p.chromium.launch()
+        browser_context = await browser.new_context()
+        page = await browser_context.new_page()
+
+        # start rendering test view
+        await page.goto(context.make_url('/frontend/rendering/'))
+        await page.wait_for_selector('#lona>div>h2:has-text("Rendering Test")')
+
+        rendering_root_element = page.locator('#lona #rendering-root')
+
+        # get total rendering steps
+        total_steps_element = page.locator('#lona #step-label #total')
+        total_steps = int(await total_steps_element.inner_text())
```

Comment:
It's better to use expected value instead of relying to UI. For example test won't fail if there will be `0` in the page.
```suggestion
        total_steps = 21
```

Reply:
I totally agree. Fixed.